lib/matplotlib/pyplot.py

```diff
@@ -1943,6 +1943,9 @@ def colormaps():
                     originally from the Neuroimaging in Python project
       terrain       mapmaker's colors, blue-green-yellow-brown-white,
                     originally from IGOR Pro
+      turbo         a spectral map (purple-blue-green-yellow-orange-red) with
+	                a bright center and darker endpoints. a smoother
```

Comment:
Can you fix the capitalization and indenting here?

Reply:
Done! Sorry capitalization style wasn't clear from the above :)